### ISSUE TITLE
LinesPerLogFile server setting doesn't work

### ISSUE THREAD

**Issue by JoneKone** [Regalis11/Barotrauma#6455]
- [ x]  To the best of my ability I have searched the issue tracker to check if the issue has already been reported.

**Description**
Logs, seem to have a hard file size limit of 60 KB and then they abruply cut off and when they do it won't instantly continue on the next file.

**Steps To Reproduce**
setup a linux server let it run with random things and see when the file reaches 60KB then let it continue and check the new files latest timestamp.

**Version**
v0.14.6.0

**Additional information**
makes it hard to follow the logs, (suggestion) please make it round based and let the log swell also if possible make it 24h standard none of this 12h... tangled stuff also the date shoudn't be on the line the time suffices as the file now has the date and it would be session based and not size based.

**Comment by Regalis11** [comment]
There's a limit to the number of lines per log file that can be adjusted using the attribute `linesperlogfile="xxx"` in the server settings file. I added the attribute to the default server settings file in https://github.com/Regalis11/Barotrauma-development/commit/02f5615d074741064b631c72f54d00bb51fa26ec to make it easier to find.

**Comment by JoneKone** [comment]
![Screenshot from 2021-08-23 20-29-36](https://user-images.githubusercontent.com/1114115/130491097-42849f54-b0a4-408a-87dd-cb3d88ba2532.png)

well it's not working or i did something wrong.

**Comment by Regalis11** [comment]
Oh, you're right! Looks like the value didn't get correctly loaded from the settings file. Fixed in https://github.com/Regalis11/Barotrauma-development/commit/4a1e66ff816974c38d958d24b4360ed862feb39b

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
